src/client/scripts/cardSubmitArea.js

```diff
@@ -5,8 +5,12 @@ Template.cardSubmitArea.events({
 
     "submit #card": function(eve){
         eve.preventDefault();
+        var author;
 
-        var author = eve.target.author.value;
+        if(!Meteor.user())
```

Comment:
Perfect use for ternary
`author = Meteor.user() ? Meteor.user() : eve.target.author.value`